### DISCUSSION TITLE
v7.13.2

### DISCUSSION THREAD

**Discussion by mvdwetering** [mvdwetering/yamaha_ynca#358] [General]
## What's Changed

* Fix remote entity volume up/down codes by @trvrnrth
* Update README.md, add RX-V4A, and RX-A730. Thanks to everyone reporting unlisted but working receivers 👍

## New Contributors
* @trvrnrth made their first contribution in https://github.com/mvdwetering/yamaha_ynca/pull/356

**Full Changelog**: https://github.com/mvdwetering/yamaha_ynca/compare/v7.13.1...v7.13.2

<hr /><em>This discussion was created from the release <a href='https://github.com/mvdwetering/yamaha_ynca/releases/tag/v7.13.2'>v7.13.2</a>.</em>

**Comment by tramperb** [comment]
I'm puzzled by the v7.13.2 update https://github.com/mvdwetering/yamaha_ynca/pull/356 which 'fixes' an issue where vol up/down codes are reversed.

For me (RX-A740), the vol+ and vol- buttons (on the media player card) work as expected.  Presumably, if I update, the fix will actually reverse the vol buttons operation (and cause the issue the fix is fixing!).

**Reply by trvrnrth** [reply]
The fix is for the remote entity commands/actions. The media player entity actions were correct and are not impacted.

**Comment by tramperb** [comment]
Aaah, thanks for that - explains why I hadn't seen it as a bug (not using the remote entity).
